obs-server.sls

```diff
@@ -1,4 +1,21 @@
-install obs server packages:
+/etc/apt/sources.list:
+  file.managed:
+    - source: salt://obs-server/sources.list
+    - user: root
+    - group: root
+    - mode: 644
+
+obs:
+  host.present:
+    - ip: 127.0.0.1
+
+refresh_packages_db:
```

Comment:
This isn't necessary, add a repo the salt way will do it
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkgrepo.html

Reply:
ping?